src/index.js

```diff
@@ -1,17 +1,93 @@
 import './style.css';
-import {
-  render,
-  displayCheckbox,
-  checkCheckbox,
-} from './checkbox.js';
-
-window.addEventListener('load', () => {
-  render();
-  displayCheckbox();
-  const checkbox = document.querySelectorAll('.checkbox');
-  for (let i = 0; i < checkbox.length; i += 1) {
-    checkbox[i].addEventListener('change', () => {
-      checkCheckbox(i);
+import Checkbox from './checkbox.js';
+import Storage from './inputRecieve.js';
+import Tasks from './task.js';
+
+const tasksDisplay = document.querySelector('.tasks');
+
+class Ui {
```

Comment:
- [ ] One of the project's requirements is implementing a function for editing task descriptions. Currently, you don't have this function and I cannot edit the description of a to-do item. Kindly add this function.  :pray:

Reply:
This issue has been addressed well. Great job! :clap: